pkgs/dart_tooling_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -220,8 +225,10 @@ base mixin DartToolingDaemonSupport
           await updateActiveVmServices();
         case 'debugSessionStopped':
           await activeVmServices
-              .remove((e.data['debugSession'] as DebugSession).id)
+              .remove(e.data['debugSessionId'] as String)
```

Comment:
This is an unrelated bug fix that I snuck in here, I can separate it out if we want